src/PlusServer/vtkPlusCommandProcessor.cxx

```diff
@@ -38,6 +38,7 @@ See License.txt for details.
 #include "vtkPlusStartStopRecordingCommand.h"
 #include "vtkPlusUpdateTransformCommand.h"
 #include "vtkPlusVersionCommand.h"
+#include "vtkPlusWinProbeCommand.h"

```

Comment:
When building the [PlusBuild](https://github.com/PlusToolkit/PlusBuild) project pointing to this PlusLib branch and WITHOUT WinProbe device support there are linking errors and the project fails to build successfully. This include of `vtkPlusWinProbeCommand.h` likely causes this error because the `vtkPlusWinProbeCommand` class contains  `#include "WinProbe/vtkPlusWinProbeVideoSource.h"`

```
8>  vtkPlusWinProbeCommand.cxx
8>  Generating Code...
8>     Creating library C:/P/PR22/lib/Release/vtkPlusServer.lib and object C:/P/PR22/lib/Release/vtkPlusServer.exp
8>vtkPlusWinProbeCommand.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: enum igsioStatus __cdecl vtkPlusWinProbeVideoSource::SetSSDecimation(unsigned char)" (__imp_?SetSSDecimation@vtkPlusWinProbeVideoSource@@QEAA?AW4igsioStatus@@E@Z) referenced in function "public: virtual enum igsioStatus __cdecl vtkPlusWinProbeCommand::Execute(void)" (?Execute@vtkPlusWinProbeCommand@@UEAA?AW4igsioStatus@@XZ) [C:\P\PR22\PlusLib-bin\src\PlusServer\vtkPlusServer.vcxproj]
8>vtkPlusWinProbeCommand.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: double __cdecl vtkPlusWinProbeVideoSource::GetTimeGainCompensation(int)" (__imp_?GetTimeGainCompensation@vtkPlusWinProbeVideoSource@@QEAANH@Z) referenced in function "public: virtual enum igsioStatus __cdecl vtkPlusWinProbeCommand::Execute(void)" (?Execute@vtkPlusWinProbeCommand@@UEAA?AW4igsioStatus@@XZ) [C:\P\PR22\PlusLib-bin\src\PlusServer\vtkPlusServer.vcxproj]
8>    Hint on symbols that are defined and could potentially match:
8>      "__declspec(dllimport) public: class std::vector<double,class std::allocator<double> > __cdecl vtkPlusUsImagingParameters::GetTimeGainCompensation(void)const " (__imp_?GetTimeGainCompensation@vtkPlusUsImagingParameters@@QEBA?AV?$vector@NV?$allocator@N@std@@@std@@XZ)
8>vtkPlusWinProbeCommand.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: enum igsioStatus __cdecl vtkPlusWinProbeVideoSource::SetTimeGainCompensation(int,double)" (__imp_?SetTimeGainCompensation@vtkPlusWinProbeVideoSource@@QEAA?AW4igsioStatus@@HN@Z) referenced in function "public: virtual enum igsioStatus __cdecl vtkPlusWinProbeCommand::Execute(void)" (?Execute@vtkPlusWinProbeCommand@@UEAA?AW4igsioStatus@@XZ) [C:\P\PR22\PlusLib-bin\src\PlusServer\vtkPlusServer.vcxproj]
8>    Hint on symbols that are defined and could potentially match:
8>      "__declspec(dllimport) public: enum igsioStatus __cdecl vtkPlusUsImagingParameters::SetTimeGainCompensation(class std::vector<double,class std::allocator<double> > const &)" (__imp_?SetTimeGainCompensation@vtkPlusUsImagingParameters@@QEAA?AW4igsioStatus@@AEBV?$vector@NV?$allocator@N@std@@@std@@@Z)
8>vtkPlusWinProbeCommand.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: double __cdecl vtkPlusWinProbeVideoSource::GetFirstGainValue(void)" (__imp_?GetFirstGainValue@vtkPlusWinProbeVideoSource@@QEAANXZ) referenced in function "public: virtual enum igsioStatus __cdecl vtkPlusWinProbeCommand::Execute(void)" (?Execute@vtkPlusWinProbeCommand@@UEAA?AW4igsioStatus@@XZ) [C:\P\PR22\PlusLib-bin\src\PlusServer\vtkPlusServer.vcxproj]
8>vtkPlusWinProbeCommand.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: enum igsioStatus __cdecl vtkPlusWinProbeVideoSource::SetFirstGainValue(double)" (__imp_?SetFirstGainValue@vtkPlusWinProbeVideoSource@@QEAA?AW4igsioStatus@@N@Z) referenced in function "public: virtual enum igsioStatus __cdecl vtkPlusWinProbeCommand::Execute(void)" (?Execute@vtkPlusWinProbeCommand@@UEAA?AW4igsioStatus@@XZ) [C:\P\PR22\PlusLib-bin\src\PlusServer\vtkPlusServer.vcxproj]
8>vtkPlusWinProbeCommand.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: double __cdecl vtkPlusWinProbeVideoSource::GetOverallTimeGainCompensation(void)" (__imp_?GetOverallTimeGainCompensation@vtkPlusWinProbeVideoSource@@QEAANXZ) referenced in function "public: virtual enum igsioStatus __cdecl vtkPlusWinProbeCommand::Execute(void)" (?Execute@vtkPlusWinProbeCommand@@UEAA?AW4igsioStatus@@XZ) [C:\P\PR22\PlusLib-bin\src\PlusServer\vtkPlusServer.vcxproj]
8>vtkPlusWinProbeCommand.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: enum igsioStatus __cdecl vtkPlusWinProbeVideoSource::SetOverallTimeGainCompensation(double)" (__imp_?SetOverallTimeGainCompensation@vtkPlusWinProbeVideoSource@@QEAA?AW4igsioStatus@@N@Z) referenced in function "public: virtual enum igsioStatus __cdecl vtkPlusWinProbeCommand::Execute(void)" (?Execute@vtkPlusWinProbeCommand@@UEAA?AW4igsioStatus@@XZ) [C:\P\PR22\PlusLib-bin\src\PlusServer\vtkPlusServer.vcxproj]
8>vtkPlusWinProbeCommand.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: float __cdecl vtkPlusWinProbeVideoSource::GetFocalPointDepth(int)" (__imp_?GetFocalPointDepth@vtkPlusWinProbeVideoSource@@QEAAMH@Z) referenced in function "public: virtual enum igsioStatus __cdecl vtkPlusWinProbeCommand::Execute(void)" (?Execute@vtkPlusWinProbeCommand@@UEAA?AW4igsioStatus@@XZ) [C:\P\PR22\PlusLib-bin\src\PlusServer\vtkPlusServer.vcxproj]
8>vtkPlusWinProbeCommand.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: enum igsioStatus __cdecl vtkPlusWinProbeVideoSource::SetFocalPointDepth(int,float)" (__imp_?SetFocalPointDepth@vtkPlusWinProbeVideoSource@@QEAA?AW4igsioStatus@@HM@Z) referenced in function "public: virtual enum igsioStatus __cdecl vtkPlusWinProbeCommand::Execute(void)" (?Execute@vtkPlusWinProbeCommand@@UEAA?AW4igsioStatus@@XZ) [C:\P\PR22\PlusLib-bin\src\PlusServer\vtkPlusServer.vcxproj]
8>vtkPlusWinProbeCommand.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: float __cdecl vtkPlusWinProbeVideoSource::GetARFIFocalPointDepth(int)" (__imp_?GetARFIFocalPointDepth@vtkPlusWinProbeVideoSource@@QEAAMH@Z) referenced in function "public: virtual enum igsioStatus __cdecl vtkPlusWinProbeCommand::Execute(void)" (?Execute@vtkPlusWinProbeCommand@@UEAA?AW4igsioStatus@@XZ) [C:\P\PR22\PlusLib-bin\src\PlusServer\vtkPlusServer.vcxproj]
8>vtkPlusWinProbeCommand.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: enum igsioStatus __cdecl vtkPlusWinProbeVideoSource::SetARFIFocalPointDepth(int,float)" (__imp_?SetARFIFocalPointDepth@vtkPlusWinProbeVideoSource@@QEAA?AW4igsioStatus@@HM@Z) referenced in function "public: virtual enum igsioStatus __cdecl vtkPlusWinProbeCommand::Execute(void)" (?Execute@vtkPlusWinProbeCommand@@UEAA?AW4igsioStatus@@XZ) [C:\P\PR22\PlusLib-bin\src\PlusServer\vtkPlusServer.vcxproj]
8>vtkPlusWinProbeCommand.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: int __cdecl vtkPlusWinProbeVideoSource::GetBMultiFocalZoneCount(void)" (__imp_?GetBMultiFocalZoneCount@vtkPlusWinProbeVideoSource@@QEAAHXZ) referenced in function "public: virtual enum igsioStatus __cdecl vtkPlusWinProbeCommand::Execute(void)" (?Execute@vtkPlusWinProbeCommand@@UEAA?AW4igsioStatus@@XZ) [C:\P\PR22\PlusLib-bin\src\PlusServer\vtkPlusServer.vcxproj]
8>vtkPlusWinProbeCommand.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: enum igsioStatus __cdecl vtkPlusWinProbeVideoSource::SetBMultiFocalZoneCount(int)" (__imp_?SetBMultiFocalZoneCount@vtkPlusWinProbeVideoSource@@QEAA?AW4igsioStatus@@H@Z) referenced in function "public: virtual enum igsioStatus __cdecl vtkPlusWinProbeCommand::Execute(void)" (?Execute@vtkPlusWinProbeCommand@@UEAA?AW4igsioStatus@@XZ) [C:\P\PR22\PlusLib-bin\src\PlusServer\vtkPlusServer.vcxproj]
8>vtkPlusWinProbeCommand.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: enum igsioStatus __cdecl vtkPlusWinProbeVideoSource::FreezeDevice(bool)" (__imp_?FreezeDevice@vtkPlusWinProbeVideoSource@@QEAA?AW4igsioStatus@@_N@Z) referenced in function "public: virtual enum igsioStatus __cdecl vtkPlusWinProbeCommand::Execute(void)" (?Execute@vtkPlusWinProbeCommand@@UEAA?AW4igsioStatus@@XZ) [C:\P\PR22\PlusLib-bin\src\PlusServer\vtkPlusServer.vcxproj]
8>vtkPlusWinProbeCommand.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: bool __cdecl vtkPlusWinProbeVideoSource::IsFrozen(void)" (__imp_?IsFrozen@vtkPlusWinProbeVideoSource@@QEAA_NXZ) referenced in function "public: virtual enum igsioStatus __cdecl vtkPlusWinProbeCommand::Execute(void)" (?Execute@vtkPlusWinProbeCommand@@UEAA?AW4igsioStatus@@XZ) [C:\P\PR22\PlusLib-bin\src\PlusServer\vtkPlusServer.vcxproj]
8>vtkPlusWinProbeCommand.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: bool __cdecl vtkPlusWinProbeVideoSource::IsScanning(void)" (__imp_?IsScanning@vtkPlusWinProbeVideoSource@@QEAA_NXZ) referenced in function "public: virtual enum igsioStatus __cdecl vtkPlusWinProbeCommand::Execute(void)" (?Execute@vtkPlusWinProbeCommand@@UEAA?AW4igsioStatus@@XZ) [C:\P\PR22\PlusLib-bin\src\PlusServer\vtkPlusServer.vcxproj]
8>vtkPlusWinProbeCommand.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > __cdecl vtkPlusWinProbeVideoSource::GetFPGARevDateString(void)" (__imp_?GetFPGARevDateString@vtkPlusWinProbeVideoSource@@QEAA?AV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@XZ) referenced in function "public: virtual enum igsioStatus __cdecl vtkPlusWinProbeCommand::Execute(void)" (?Execute@vtkPlusWinProbeCommand@@UEAA?AW4igsioStatus@@XZ) [C:\P\PR22\PlusLib-bin\src\PlusServer\vtkPlusServer.vcxproj]
8>vtkPlusWinProbeCommand.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: void __cdecl vtkPlusWinProbeVideoSource::SetSpatialCompoundEnabled(bool)" (__imp_?SetSpatialCompoundEnabled@vtkPlusWinProbeVideoSource@@QEAAX_N@Z) referenced in function "public: virtual enum igsioStatus __cdecl vtkPlusWinProbeCommand::Execute(void)" (?Execute@vtkPlusWinProbeCommand@@UEAA?AW4igsioStatus@@XZ) [C:\P\PR22\PlusLib-bin\src\PlusServer\vtkPlusServer.vcxproj]
8>vtkPlusWinProbeCommand.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: bool __cdecl vtkPlusWinProbeVideoSource::GetSpatialCompoundEnabled(void)" (__imp_?GetSpatialCompoundEnabled@vtkPlusWinProbeVideoSource@@QEAA_NXZ) referenced in function "public: virtual enum igsioStatus __cdecl vtkPlusWinProbeCommand::Execute(void)" (?Execute@vtkPlusWinProbeCommand@@UEAA?AW4igsioStatus@@XZ) [C:\P\PR22\PlusLib-bin\src\PlusServer\vtkPlusServer.vcxproj]
8>vtkPlusWinProbeCommand.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: void __cdecl vtkPlusWinProbeVideoSource::SetSpatialCompoundCount(int)" (__imp_?SetSpatialCompoundCount@vtkPlusWinProbeVideoSource@@QEAAXH@Z) referenced in function "public: virtual enum igsioStatus __cdecl vtkPlusWinProbeCommand::Execute(void)" (?Execute@vtkPlusWinProbeCommand@@UEAA?AW4igsioStatus@@XZ) [C:\P\PR22\PlusLib-bin\src\PlusServer\vtkPlusServer.vcxproj]
8>vtkPlusWinProbeCommand.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: int __cdecl vtkPlusWinProbeVideoSource::GetSpatialCompoundCount(void)" (__imp_?GetSpatialCompoundCount@vtkPlusWinProbeVideoSource@@QEAAHXZ) referenced in function "public: virtual enum igsioStatus __cdecl vtkPlusWinProbeCommand::Execute(void)" (?Execute@vtkPlusWinProbeCommand@@UEAA?AW4igsioStatus@@XZ) [C:\P\PR22\PlusLib-bin\src\PlusServer\vtkPlusServer.vcxproj]
8>vtkPlusWinProbeCommand.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: float __cdecl vtkPlusWinProbeVideoSource::GetSpatialCompoundAngle(void)" (__imp_?GetSpatialCompoundAngle@vtkPlusWinProbeVideoSource@@QEAAMXZ) referenced in function "public: virtual enum igsioStatus __cdecl vtkPlusWinProbeCommand::Execute(void)" (?Execute@vtkPlusWinProbeCommand@@UEAA?AW4igsioStatus@@XZ) [C:\P\PR22\PlusLib-bin\src\PlusServer\vtkPlusServer.vcxproj]
8>vtkPlusWinProbeCommand.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: void __cdecl vtkPlusWinProbeVideoSource::SetBHarmonicEnabled(bool)" (__imp_?SetBHarmonicEnabled@vtkPlusWinProbeVideoSource@@QEAAX_N@Z) referenced in function "public: virtual enum igsioStatus __cdecl vtkPlusWinProbeCommand::Execute(void)" (?Execute@vtkPlusWinProbeCommand@@UEAA?AW4igsioStatus@@XZ) [C:\P\PR22\PlusLib-bin\src\PlusServer\vtkPlusServer.vcxproj]
8>vtkPlusWinProbeCommand.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: bool __cdecl vtkPlusWinProbeVideoSource::GetBHarmonicEnabled(void)" (__imp_?GetBHarmonicEnabled@vtkPlusWinProbeVideoSource@@QEAA_NXZ) referenced in function "public: virtual enum igsioStatus __cdecl vtkPlusWinProbeCommand::Execute(void)" (?Execute@vtkPlusWinProbeCommand@@UEAA?AW4igsioStatus@@XZ) [C:\P\PR22\PlusLib-bin\src\PlusServer\vtkPlusServer.vcxproj]
8>vtkPlusWinProbeCommand.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: void __cdecl vtkPlusWinProbeVideoSource::SetBRFEnabled(bool)" (__imp_?SetBRFEnabled@vtkPlusWinProbeVideoSource@@QEAAX_N@Z) referenced in function "public: virtual enum igsioStatus __cdecl vtkPlusWinProbeCommand::Execute(void)" (?Execute@vtkPlusWinProbeCommand@@UEAA?AW4igsioStatus@@XZ) [C:\P\PR22\PlusLib-bin\src\PlusServer\vtkPlusServer.vcxproj]
8>vtkPlusWinProbeCommand.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: void __cdecl vtkPlusWinProbeVideoSource::SetMModeEnabled(bool)" (__imp_?SetMModeEnabled@vtkPlusWinProbeVideoSource@@QEAAX_N@Z) referenced in function "public: virtual enum igsioStatus __cdecl vtkPlusWinProbeCommand::Execute(void)" (?Execute@vtkPlusWinProbeCommand@@UEAA?AW4igsioStatus@@XZ) [C:\P\PR22\PlusLib-bin\src\PlusServer\vtkPlusServer.vcxproj]
8>vtkPlusWinProbeCommand.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: bool __cdecl vtkPlusWinProbeVideoSource::GetMModeEnabled(void)" (__imp_?GetMModeEnabled@vtkPlusWinProbeVideoSource@@QEAA_NXZ) referenced in function "public: virtual enum igsioStatus __cdecl vtkPlusWinProbeCommand::Execute(void)" (?Execute@vtkPlusWinProbeCommand@@UEAA?AW4igsioStatus@@XZ) [C:\P\PR22\PlusLib-bin\src\PlusServer\vtkPlusServer.vcxproj]
8>vtkPlusWinProbeCommand.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: void __cdecl vtkPlusWinProbeVideoSource::SetMPRFrequency(int)" (__imp_?SetMPRFrequency@vtkPlusWinProbeVideoSource@@QEAAXH@Z) referenced in function "public: virtual enum igsioStatus __cdecl vtkPlusWinProbeCommand::Execute(void)" (?Execute@vtkPlusWinProbeCommand@@UEAA?AW4igsioStatus@@XZ) [C:\P\PR22\PlusLib-bin\src\PlusServer\vtkPlusServer.vcxproj]
8>vtkPlusWinProbeCommand.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: int __cdecl vtkPlusWinProbeVideoSource::GetMPRFrequency(void)" (__imp_?GetMPRFrequency@vtkPlusWinProbeVideoSource@@QEAAHXZ) referenced in function "public: virtual enum igsioStatus __cdecl vtkPlusWinProbeCommand::Execute(void)" (?Execute@vtkPlusWinProbeCommand@@UEAA?AW4igsioStatus@@XZ) [C:\P\PR22\PlusLib-bin\src\PlusServer\vtkPlusServer.vcxproj]
8>vtkPlusWinProbeCommand.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: void __cdecl vtkPlusWinProbeVideoSource::SetMLineIndex(int)" (__imp_?SetMLineIndex@vtkPlusWinProbeVideoSource@@QEAAXH@Z) referenced in function "public: virtual enum igsioStatus __cdecl vtkPlusWinProbeCommand::Execute(void)" (?Execute@vtkPlusWinProbeCommand@@UEAA?AW4igsioStatus@@XZ) [C:\P\PR22\PlusLib-bin\src\PlusServer\vtkPlusServer.vcxproj]
8>vtkPlusWinProbeCommand.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: int __cdecl vtkPlusWinProbeVideoSource::GetMLineIndex(void)" (__imp_?GetMLineIndex@vtkPlusWinProbeVideoSource@@QEAAHXZ) referenced in function "public: virtual enum igsioStatus __cdecl vtkPlusWinProbeCommand::Execute(void)" (?Execute@vtkPlusWinProbeCommand@@UEAA?AW4igsioStatus@@XZ) [C:\P\PR22\PlusLib-bin\src\PlusServer\vtkPlusServer.vcxproj]
8>vtkPlusWinProbeCommand.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: void __cdecl vtkPlusWinProbeVideoSource::SetMWidth(int)" (__imp_?SetMWidth@vtkPlusWinProbeVideoSource@@QEAAXH@Z) referenced in function "public: virtual enum igsioStatus __cdecl vtkPlusWinProbeCommand::Execute(void)" (?Execute@vtkPlusWinProbeCommand@@UEAA?AW4igsioStatus@@XZ) [C:\P\PR22\PlusLib-bin\src\PlusServer\vtkPlusServer.vcxproj]
8>vtkPlusWinProbeCommand.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: int __cdecl vtkPlusWinProbeVideoSource::GetMWidth(void)" (__imp_?GetMWidth@vtkPlusWinProbeVideoSource@@QEAAHXZ) referenced in function "public: virtual enum igsioStatus __cdecl vtkPlusWinProbeCommand::Execute(void)" (?Execute@vtkPlusWinProbeCommand@@UEAA?AW4igsioStatus@@XZ) [C:\P\PR22\PlusLib-bin\src\PlusServer\vtkPlusServer.vcxproj]
8>vtkPlusWinProbeCommand.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: void __cdecl vtkPlusWinProbeVideoSource::SetMAcousticLineCount(int)" (__imp_?SetMAcousticLineCount@vtkPlusWinProbeVideoSource@@QEAAXH@Z) referenced in function "public: virtual enum igsioStatus __cdecl vtkPlusWinProbeCommand::Execute(void)" (?Execute@vtkPlusWinProbeCommand@@UEAA?AW4igsioStatus@@XZ) [C:\P\PR22\PlusLib-bin\src\PlusServer\vtkPlusServer.vcxproj]
8>vtkPlusWinProbeCommand.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: int __cdecl vtkPlusWinProbeVideoSource::GetMAcousticLineCount(void)" (__imp_?GetMAcousticLineCount@vtkPlusWinProbeVideoSource@@QEAAHXZ) referenced in function "public: virtual enum igsioStatus __cdecl vtkPlusWinProbeCommand::Execute(void)" (?Execute@vtkPlusWinProbeCommand@@UEAA?AW4igsioStatus@@XZ) [C:\P\PR22\PlusLib-bin\src\PlusServer\vtkPlusServer.vcxproj]
8>vtkPlusWinProbeCommand.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: void __cdecl vtkPlusWinProbeVideoSource::SetMDepth(int)" (__imp_?SetMDepth@vtkPlusWinProbeVideoSource@@QEAAXH@Z) referenced in function "public: virtual enum igsioStatus __cdecl vtkPlusWinProbeCommand::Execute(void)" (?Execute@vtkPlusWinProbeCommand@@UEAA?AW4igsioStatus@@XZ) [C:\P\PR22\PlusLib-bin\src\PlusServer\vtkPlusServer.vcxproj]
8>vtkPlusWinProbeCommand.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: int __cdecl vtkPlusWinProbeVideoSource::GetMDepth(void)" (__imp_?GetMDepth@vtkPlusWinProbeVideoSource@@QEAAHXZ) referenced in function "public: virtual enum igsioStatus __cdecl vtkPlusWinProbeCommand::Execute(void)" (?Execute@vtkPlusWinProbeCommand@@UEAA?AW4igsioStatus@@XZ) [C:\P\PR22\PlusLib-bin\src\PlusServer\vtkPlusServer.vcxproj]
8>vtkPlusWinProbeCommand.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: void __cdecl vtkPlusWinProbeVideoSource::SetBFrameRateLimit(int)" (__imp_?SetBFrameRateLimit@vtkPlusWinProbeVideoSource@@QEAAXH@Z) referenced in function "public: virtual enum igsioStatus __cdecl vtkPlusWinProbeCommand::Execute(void)" (?Execute@vtkPlusWinProbeCommand@@UEAA?AW4igsioStatus@@XZ) [C:\P\PR22\PlusLib-bin\src\PlusServer\vtkPlusServer.vcxproj]
8>vtkPlusWinProbeCommand.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: int __cdecl vtkPlusWinProbeVideoSource::GetBFrameRateLimit(void)" (__imp_?GetBFrameRateLimit@vtkPlusWinProbeVideoSource@@QEAAHXZ) referenced in function "public: virtual enum igsioStatus __cdecl vtkPlusWinProbeCommand::Execute(void)" (?Execute@vtkPlusWinProbeCommand@@UEAA?AW4igsioStatus@@XZ) [C:\P\PR22\PlusLib-bin\src\PlusServer\vtkPlusServer.vcxproj]
8>vtkPlusWinProbeCommand.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: void __cdecl vtkPlusWinProbeVideoSource::SetARFIEnabled(bool)" (__imp_?SetARFIEnabled@vtkPlusWinProbeVideoSource@@QEAAX_N@Z) referenced in function "public: virtual enum igsioStatus __cdecl vtkPlusWinProbeCommand::Execute(void)" (?Execute@vtkPlusWinProbeCommand@@UEAA?AW4igsioStatus@@XZ) [C:\P\PR22\PlusLib-bin\src\PlusServer\vtkPlusServer.vcxproj]
8>vtkPlusWinProbeCommand.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: bool __cdecl vtkPlusWinProbeVideoSource::GetARFIEnabled(void)" (__imp_?GetARFIEnabled@vtkPlusWinProbeVideoSource@@QEAA_NXZ) referenced in function "public: virtual enum igsioStatus __cdecl vtkPlusWinProbeCommand::Execute(void)" (?Execute@vtkPlusWinProbeCommand@@UEAA?AW4igsioStatus@@XZ) [C:\P\PR22\PlusLib-bin\src\PlusServer\vtkPlusServer.vcxproj]
8>vtkPlusWinProbeCommand.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: void __cdecl vtkPlusWinProbeVideoSource::SetARFIStartSample(int)" (__imp_?SetARFIStartSample@vtkPlusWinProbeVideoSource@@QEAAXH@Z) referenced in function "public: virtual enum igsioStatus __cdecl vtkPlusWinProbeCommand::Execute(void)" (?Execute@vtkPlusWinProbeCommand@@UEAA?AW4igsioStatus@@XZ) [C:\P\PR22\PlusLib-bin\src\PlusServer\vtkPlusServer.vcxproj]
8>vtkPlusWinProbeCommand.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: int __cdecl vtkPlusWinProbeVideoSource::GetARFIStartSample(void)" (__imp_?GetARFIStartSample@vtkPlusWinProbeVideoSource@@QEAAHXZ) referenced in function "public: virtual enum igsioStatus __cdecl vtkPlusWinProbeCommand::Execute(void)" (?Execute@vtkPlusWinProbeCommand@@UEAA?AW4igsioStatus@@XZ) [C:\P\PR22\PlusLib-bin\src\PlusServer\vtkPlusServer.vcxproj]
8>vtkPlusWinProbeCommand.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: void __cdecl vtkPlusWinProbeVideoSource::SetARFIStopSample(int)" (__imp_?SetARFIStopSample@vtkPlusWinProbeVideoSource@@QEAAXH@Z) referenced in function "public: virtual enum igsioStatus __cdecl vtkPlusWinProbeCommand::Execute(void)" (?Execute@vtkPlusWinProbeCommand@@UEAA?AW4igsioStatus@@XZ) [C:\P\PR22\PlusLib-bin\src\PlusServer\vtkPlusServer.vcxproj]
8>vtkPlusWinProbeCommand.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: int __cdecl vtkPlusWinProbeVideoSource::GetARFIStopSample(void)" (__imp_?GetARFIStopSample@vtkPlusWinProbeVideoSource@@QEAAHXZ) referenced in function "public: virtual enum igsioStatus __cdecl vtkPlusWinProbeCommand::Execute(void)" (?Execute@vtkPlusWinProbeCommand@@UEAA?AW4igsioStatus@@XZ) [C:\P\PR22\PlusLib-bin\src\PlusServer\vtkPlusServer.vcxproj]
8>vtkPlusWinProbeCommand.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: enum igsioStatus __cdecl vtkPlusWinProbeVideoSource::SetARFIPrePushLineRepeatCount(int)" (__imp_?SetARFIPrePushLineRepeatCount@vtkPlusWinProbeVideoSource@@QEAA?AW4igsioStatus@@H@Z) referenced in function "public: virtual enum igsioStatus __cdecl vtkPlusWinProbeCommand::Execute(void)" (?Execute@vtkPlusWinProbeCommand@@UEAA?AW4igsioStatus@@XZ) [C:\P\PR22\PlusLib-bin\src\PlusServer\vtkPlusServer.vcxproj]
8>vtkPlusWinProbeCommand.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: int __cdecl vtkPlusWinProbeVideoSource::GetARFIPrePushLineRepeatCount(void)" (__imp_?GetARFIPrePushLineRepeatCount@vtkPlusWinProbeVideoSource@@QEAAHXZ) referenced in function "public: virtual enum igsioStatus __cdecl vtkPlusWinProbeCommand::Execute(void)" (?Execute@vtkPlusWinProbeCommand@@UEAA?AW4igsioStatus@@XZ) [C:\P\PR22\PlusLib-bin\src\PlusServer\vtkPlusServer.vcxproj]
8>vtkPlusWinProbeCommand.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: enum igsioStatus __cdecl vtkPlusWinProbeVideoSource::SetARFIPostPushLineRepeatCount(int)" (__imp_?SetARFIPostPushLineRepeatCount@vtkPlusWinProbeVideoSource@@QEAA?AW4igsioStatus@@H@Z) referenced in function "public: virtual enum igsioStatus __cdecl vtkPlusWinProbeCommand::Execute(void)" (?Execute@vtkPlusWinProbeCommand@@UEAA?AW4igsioStatus@@XZ) [C:\P\PR22\PlusLib-bin\src\PlusServer\vtkPlusServer.vcxproj]
8>vtkPlusWinProbeCommand.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: int __cdecl vtkPlusWinProbeVideoSource::GetARFIPostPushLineRepeatCount(void)" (__imp_?GetARFIPostPushLineRepeatCount@vtkPlusWinProbeVideoSource@@QEAAHXZ) referenced in function "public: virtual enum igsioStatus __cdecl vtkPlusWinProbeCommand::Execute(void)" (?Execute@vtkPlusWinProbeCommand@@UEAA?AW4igsioStatus@@XZ) [C:\P\PR22\PlusLib-bin\src\PlusServer\vtkPlusServer.vcxproj]
8>vtkPlusWinProbeCommand.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: int __cdecl vtkPlusWinProbeVideoSource::GetTransducerInternalID(void)" (__imp_?GetTransducerInternalID@vtkPlusWinProbeVideoSource@@QEAAHXZ) referenced in function "public: virtual enum igsioStatus __cdecl vtkPlusWinProbeCommand::Execute(void)" (?Execute@vtkPlusWinProbeCommand@@UEAA?AW4igsioStatus@@XZ) [C:\P\PR22\PlusLib-bin\src\PlusServer\vtkPlusServer.vcxproj]
8>vtkPlusWinProbeCommand.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: void __cdecl vtkPlusWinProbeVideoSource::SetARFIPushConfigurationString(class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> >)" (__imp_?SetARFIPushConfigurationString@vtkPlusWinProbeVideoSource@@QEAAXV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@@Z) referenced in function "public: virtual enum igsioStatus __cdecl vtkPlusWinProbeCommand::Execute(void)" (?Execute@vtkPlusWinProbeCommand@@UEAA?AW4igsioStatus@@XZ) [C:\P\PR22\PlusLib-bin\src\PlusServer\vtkPlusServer.vcxproj]
8>vtkPlusWinProbeCommand.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > __cdecl vtkPlusWinProbeVideoSource::GetARFIPushConfigurationString(void)" (__imp_?GetARFIPushConfigurationString@vtkPlusWinProbeVideoSource@@QEAA?AV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@XZ) referenced in function "public: virtual enum igsioStatus __cdecl vtkPlusWinProbeCommand::Execute(void)" (?Execute@vtkPlusWinProbeCommand@@UEAA?AW4igsioStatus@@XZ) [C:\P\PR22\PlusLib-bin\src\PlusServer\vtkPlusServer.vcxproj]
8>vtkPlusWinProbeCommand.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: enum igsioStatus __cdecl vtkPlusWinProbeVideoSource::SendCommand(char const *)" (__imp_?SendCommand@vtkPlusWinProbeVideoSource@@QEAA?AW4igsioStatus@@PEBD@Z) referenced in function "public: virtual enum igsioStatus __cdecl vtkPlusWinProbeCommand::Execute(void)" (?Execute@vtkPlusWinProbeCommand@@UEAA?AW4igsioStatus@@XZ) [C:\P\PR22\PlusLib-bin\src\PlusServer\vtkPlusServer.vcxproj]
8>C:\P\PR22\bin\Release\vtkPlusServer.dll : fatal error LNK1120: 53 unresolved externals [C:\P\PR22\PlusLib-bin\src\PlusServer\vtkPlusServer.vcxproj]
```

Reply:
updating this statement to be inside of an #ifdef statement does not appear to be the issue. I tested this by building pluslib with the repo pointing towards this repo and the commit at the newest commit (part 2). All of these errors are still showing for me.

Reply:
https://github.com/hannah-morilak/PlusLib/pull/1#discussion_r1092219448 may be the change that you are missing